encord/project.py

```diff
@@ -958,3 +968,44 @@ def create_bundle(self) -> Bundle:
         See the :class:`encord.http.bundle.Bundle` documentation for more details
         """
         return Bundle()
+
+    def list_collaborator_timers(
+        self,
+        after: datetime.datetime,
+        before: Optional[datetime.datetime] = None,
+        group_by_data_unit: bool = True,
+    ) -> Iterable[CollaboratorTimer]:
```

Comment:
Hmm, not sure I get your point. Generator is an iterable. Iterable basically just implies that the entity implements __iter__ and __next__, which is true for Generator. 
So if we return iterable and then change it for Generator no types will be broken. At least as far as I can see, perhaps I am missing something? 

> Now you'd be forced to generate the entire list
I think you mixing Iterable and Sequence here. Sequence is required to support random access and provide the overall length of the collection, so it would have the problems you mentioned. While iterable seems to be fine.

Reply:
Got it, yes I believe you're right (I'd have to play with it). 

I guess I'd suggest that people get really good code examples (ideally examples as part of the doc strings!) where they are shown how they can use this function without paginating "too much".